app/models/menu_items_menu.rb

```diff
@@ -4,7 +4,6 @@ class MenuItemsMenu < ActiveRecord::Base
 
   validates :daily_stock, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validates :menu_item, presence: true
-  validates :menu, presence: true
```

Comment:
I tried to reproduce what AA does so it can be tested in console. 
Or put a `binding.pry` and see that the result is the same.

It is a little hacky, I am not comfortable with it either. We are open for alternatives/suggestions.